### PR TITLE
archs fix for tdrop and bashmount

### PR DESCRIPTION
**Pull request by TeddyDD**
xcheckmypkgs screamed at me so I fixed this to make it happy
(unfortunately xlint complains about custom variable `archs` now)

I included some fixes for tdrop template

### CODE REVIEW

File: srcpkgs/bashmount/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
-noarch=yes
-license="GPL-2"
-homepage="https://github.com/jamielinux/$pkgname"
-distfiles="$homepage/archive/$version.tar.gz"
+license="GPL-2.0-or-later"
+homepage="https://github.com/jamielinux/bashmount"
+distfiles="https://github.com/jamielinux/bashmount/archive/$version.tar.gz"
 checksum=32f328e1921f60b429e2ab389ecc314f5bd7545ac4385c7c7918d08ebe9ecc22
+archs=noarch
```
Comment (maxice8) [inline srcpkgs/bashmount/template]: should be where noarch=yes generally is

File: srcpkgs/tdrop/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-noarch=yes
+revision=3
 depends="bash xdotool xprop xwininfo"
 short_desc="Turn terminal emulator or any program into dropdown window"
 maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/noctuid/tdrop"
 distfiles="${homepage}/archive/v${version}.tar.gz"
 checksum=56d6e75fb17046d04234740b71ea10b422207420c48c3ec2d6131d19fa2a8597
+archs=noarch
```
Comment (maxice8) [inline srcpkgs/tdrop/template]: ditto

File: srcpkgs/tdrop/template [hunk 3]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'tdrop'
 pkgname=tdrop
 version=0.2
-revision=2
-noarch=yes
+revision=3
```
Comment (maxice8) [inline srcpkgs/tdrop/template]: don't bump revision when there is no user-facing changes

File: srcpkgs/bashmount/template (hunk 1 above)
Comment (TeddyDD) [reply srcpkgs/bashmount/template]: I get it, seems like xlint was not updated yet, I'll change that

**Comment by maxice8** [comment]
On Sat, Feb 16, 2019 at 02:49:03PM -0800, Daniel Lewan wrote:
> TeddyDD commented on this pull request.
> 
> 
> 
> >  checksum=32f328e1921f60b429e2ab389ecc314f5bd7545ac4385c7c7918d08ebe9ecc22
> +archs=noarch
> 
> I get it, seems like xlint was not updated yet, I'll change that
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8810#discussion_r257480612

I already submitted a PR for xlint just needs to be updated.

**Pull request merged**